.pre-commit-config.yaml

```diff
@@ -48,5 +48,13 @@ repos:
       - id: golangci-lint
         args: [--new-from-patch=/tmp/diff.patch]
 
+    # Rules are in .markdownlint.json file
+    # See https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md for rule descriptions
+  - repo: https://github.com/igorshubovych/markdownlint-cli
```

Comment:
I was looking into this a while ago and I was thinking about using https://github.com/DavidAnson/markdownlint-cli2 since it's from the same author as `markdownlint`. Did you try it? Any reason for choosing https://github.com/igorshubovych/markdownlint-cli ?

Reply:
I didn't try that but I noticed that `markdownlint-cli` applies the rules from `markdownlint`. I chose `markdownlint-cli` because that's what we used in NSM. I can give the other one a try though

Reply:
Replaced with markdownlint-cli2.